XamlControlsGallery/ControlPages/XamlUICommandPage.xaml

```diff
@@ -0,0 +1,58 @@
+﻿<Page x:Class="AppUIBasics.ControlPages.XamlUICommandPage" 
+      xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+      xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml" 
+      xmlns:local="using:AppUIBasics"
+      xmlns:local2="using:AppUIBasics.ControlPages"
+      xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+      xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+      xmlns:muxcontrols="using:Microsoft.UI.Xaml.Controls"
+      mc:Ignorable="d">
+    <Page.Resources>
+        <Style x:Key="HorizontalSwipe" TargetType="ListViewItem" BasedOn="{StaticResource ListViewItemRevealStyle}">
+            <Setter Property="Height" Value="60"/>
+            <Setter Property="Padding" Value="0"/>
+            <Setter Property="HorizontalContentAlignment" Value="Stretch"/>
+            <Setter Property="VerticalContentAlignment" Value="Stretch"/>
+            <Setter Property="BorderThickness" Value="0"/>
+        </Style>
+        <XamlUICommand x:Name="CustomXamlUICommand" ExecuteRequested="CustomXamlUICommand_ExecuteRequested" Description="This is a custom command" Label="Custom XamlUICommand">
```

Comment:
Looks like the description ("This is a custom command") is getting assigned to the AppBarButton's tooltip. Is that expected behavior? I would have thought that the Label would have been the more appropriate tooltip text.